routes/curations.js

```diff
@@ -63,9 +63,11 @@ async function updateCurations(request, response) {
   })
   if (curationErrors.length > 0) return response.status(400).send({ errors: curationErrors })
   const result = await curationService.addOrUpdate(userGithub, serviceGithub, info, request.body)
+  if (!result.contribution) return response.status(400).send({ errors: result.errors })
```

Comment:
suggest throwing in addOrUpdate catching/handling here

Reply:
We have added a try / catch method as you suggested